task/orka-full/0.1/README.md

```diff
@@ -0,0 +1,177 @@
+
+This `Task` can replace the `orka-init`, `orka-deploy`, and `orka-teardown` modular tasks when your workflow works with a single macOS build agent.
+
+## `orka-full`
+
+A `Task` that creates a VM template with the specified configuration, deploys a VM instance from it, and then cleans up the environment. All operations in this `Task` are performed against an Orka environment.
+
+### Prerequisites
+
+* You need a Kubernetes cluster with Tekton Pipelines v0.16.0 or later configured.
+* You need an Orka environment with the following components:
+  * Orka 1.4.1 or later.
+  * [An Orka service endpoint](https://orkadocs.macstadium.com/docs/endpoint-faqs#whats-the-orka-service-endpoint) (IP or custom domain). Usually, `http://10.221.188.100`, `http://10.10.10.100` or `https://<custom-domain>`.
+  * A dedicated Orka user with valid credentials (email & password). Create a new user or request one from your Orka administrator.
+  * An SSH-enabled base image and the respective SSH credentials (email & password OR SSH key). Use an [existing base image](https://orkadocs.macstadium.com/docs/existing-images-upload-management) or [create your own](https://orkadocs.macstadium.com/docs/creating-an-ssh-enabled-image). 
+* You need an active VPN connection between your Kubernetes cluster and Orka. Use a [VPN client](https://orkadocs.macstadium.com/docs/vpn-connect) for temporary access or create a [site-to-site VPN tunnel](https://orkadocs.macstadium.com/docs/aws-orka-connections) for permanent access.
+
+See also: [Using Orka, At a Glance](https://orkadocs.macstadium.com/docs/quick-start-introduction)
+
+See also: [GCP-MacStadium Site-to-Site VPN](https://docs.macstadium.com/docs/google-cloud-setup)
+
+### Installation
+
+Before you can use these `Tasks` in Tekton pipelines, you need to install them and the Orka configuration in your Kubernetes cluster.
+
+**Default namespace installation**
+
+To install in the `default` namespace of your Kubernetes cluster, run the following command against your actual Orka API endpoint.
+
+```sh
+ORKA_API=http://10.221.188.100 ./install.sh --apply
+```
+
+To uninstall from the `default` namespace, run the script with the `-d` or `--delete` flag:
+
+```sh
+./install.sh --delete
+```
+
+**Custom namespace installation**
+
+To install in a custom namespace, run the following command against your preferred namespace and your actual Orka API endpoint:
+
+```sh
+NAMESPACE=tekton-orka ORKA_API=http://10.221.188.100 ./install.sh --apply
+```
+
+To uninstall from a selected namespace, run the script with the `-d` or `--delete` flag against the namespace:
+
+```sh
+NAMESPACE=tekton-orka ./install.sh --delete
+```
+
+### Storing your credentials
+
+The provided `Tasks` look for two Kubernetes secrets that store your credentials: `orka-creds` for the Orka user and `orka-ssh-creds` for the SSH credentials. In the current setup, both secrets have `username` and `password` keys.
```

Comment:
```suggestion
The provided `Task` looks for two Kubernetes secrets that store your credentials: `orka-creds` for the Orka user and `orka-ssh-creds` for the SSH credentials. In the current setup, both secrets have `username` and `password` keys.
```

Reply:
Also, another note here ... the `orka-creds` secret has keys of `email` and `password`, while the `orka-ssh-creds` secret has keys of `username` and `password`. We will need to correct this in the original README as well, apologies for not catching this before.